plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -1140,137 +1163,137 @@ public void testNullBehavior()
 
         // Default null value for long single value columns is 0
         assertThat(query("SELECT long_col" +
-                        "  FROM " + ALL_TYPES_TABLE +
-                        "  WHERE string_col = 'array_null'"))
+                "  FROM " + ALL_TYPES_TABLE +
+                "  WHERE string_col = 'array_null'"))
```

Comment:
Could you please extract a commit for indentation changes?

Reply:
I'll revert and update shortly.